api/src/wfl/module/aou.clj

```diff
@@ -297,43 +271,6 @@
                      :uuid                    uuid
                      :analysis_version_number analysis_version_number
                      :chip_well_barcode       chip_well_barcode}))]
-          (let [inserted            (jdbc/insert-multi! tx table (map sql-rize (vals to-be-submitted)))
+          (let [_                   (jdbc/insert-multi! tx table (map sql-rize (vals to-be-submitted)))
```

Comment:
Again, move this out of the let bindings.

Reply:
thanks, will do 👍